### PR TITLE
[YUNIKORN-1019] Update gh-pages

### PR DESCRIPTION
**Pull request by craigcondit**
JIRA: https://issues.apache.org/jira/browse/YUNIKORN-1019

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -129,12 +129,12 @@ Cloudera, Linkedin, Microsoft, Nvidia, Tencent, Uber, etc. (sorted by alphabet).
...
-- When you are coding use the [Coding Guidelines](docs/coding-guidelines.md).
-- Please read [How to Contribute](docs/how-to-contribute.md) to understand the procedure and guidelines of making contributions.
-- We have periodically community sync ups in multiple timezones and languages, please find [Community Syncup](https://github.com/apache/incubator-yunikorn-core/blob/master/docs/community-sync-up.md) to attend online sync ups. 
-
+- Logging an issue or improvement use the [Reporting an issue Guide](https://yunikorn.apache.org/community/reporting_issues).
+- Before you contributing code or documentation to YuniKorn, please read our [Get Involved](https://yunikorn.apache.org/community/get_involved) guide.
+- When you are coding use the [Coding Guidelines](https://yunikorn.apache.org/community/coding_guidelines).
+- Please read [How to Contribute](https://yunikorn.apache.org/community/how_to_contribute) to understand the procedure and guidelines of making contributions.
+- We have periodically community sync ups in multiple timezones and languages, please find [Events](https://yunikorn.apache.org/community/events) to attend online sync ups.
```
Comment (yuchaoran2011) [inline index.md:136]:
```suggestion
- We have periodic community sync ups in multiple timezones and languages, please find [Events](https://yunikorn.apache.org/community/events) to attend online sync ups.
```

File: index.md [hunk 2]
```diff
@@ -129,12 +129,12 @@ Cloudera, Linkedin, Microsoft, Nvidia, Tencent, Uber, etc. (sorted by alphabet).
...
 
 To get involved, please read following resources.
-- Logging an issue or improvement use the [Reporting an issue Guide](https://github.com/apache/incubator-yunikorn-core/blob/master/docs/reporting-issues.md).
-- Before you contributing code or documentation to YuniKorn, please read our [Developer Guide](https://github.com/apache/incubator-yunikorn-core/blob/master/docs/developer-guide.md).
-- When you are coding use the [Coding Guidelines](docs/coding-guidelines.md).
-- Please read [How to Contribute](docs/how-to-contribute.md) to understand the procedure and guidelines of making contributions.
-- We have periodically community sync ups in multiple timezones and languages, please find [Community Syncup](https://github.com/apache/incubator-yunikorn-core/blob/master/docs/community-sync-up.md) to attend online sync ups. 
-
+- Logging an issue or improvement use the [Reporting an issue Guide](https://yunikorn.apache.org/community/reporting_issues).
+- Before you contributing code or documentation to YuniKorn, please read our [Get Involved](https://yunikorn.apache.org/community/get_involved) guide.
```
Comment (yuchaoran2011) [inline index.md:133]:
```suggestion
- Before contributing code or documentation to YuniKorn, please read our [Get Involved](https://yunikorn.apache.org/community/get_involved) guide.
```

**Review by yuchaoran2011 — APPROVED** [review]
Two minor nits that have been there before this PR, but will be nice to have them fixed while we are here

File: index.md [hunk 3]
```diff
@@ -99,28 +96,31 @@ like the default scheduler, in the cluster: [Co-existing with other K8s schedule
...
-| 1.17.x | 0.10.0 | ? |
-| 1.18.x | 0.10.0 | ? |
+| 1.16.x | 0.10.0 | 0.11.0 |
+| 1.17.x | 0.10.0 | 0.11.0 |
+| 1.18.x | 0.10.0 | 0.11.0 |
+| 1.19.x | 0.11.0 | - |
+| 1.20.x | 0.12.1 | - |
+| 1.21.x | 0.12.1 | - |
+| 1.22.x | 0.12.2 (upcoming) | - |
+| 1.23.x | 0.12.2 (upcoming) | - |
```
Comment (wilfred-s) [inline index.md:106]: we should never point to unreleased versions

**Review by wilfred-s — CHANGES_REQUESTED** [review]

File: index.md (hunk 3 above)
Comment (craigcondit) [reply index.md]: fixed

File: index.md (hunk 2 above)
Comment (craigcondit) [reply index.md]: fixed

File: index.md (hunk 1 above)
Comment (craigcondit) [reply index.md]: fixed

**Review by yuchaoran2011 — APPROVED** [review]

**Pull request merged**